assets/scss/helpers/_classes.scss

```diff
@@ -1,130 +1,3 @@
-.micro {
-  font-size: 11px;
-  line-height: 13px;
-  color: $dp-color-grey;
-  text-transform: uppercase;
-}
-/* TEXT COLORS - grey */
-.c-dark-grey {
-  color: $dp-color-darkgrey;
-  /* text */
-}
-
-.c-grey {
-  color: $dp-color-grey;
-  /* icons fonts */
-}
-
-.c-light-grey {
-  color: $dp-color-lightgrey;
-  /* placeholders, hovers*/
-}
-/* VERTICAL SPACES */
```

Comment:
Are these spacing classes currently not being used? This is a breaking change, and we should test it against the webapp.

Reply:
These classes are not being used, they were created in the past to visually show what the library was going to be used for.

Reply:
Awesome, @IgnacioRodrigues . Let's move forward and remove these unused styles and definitions